irida_staramr_results/api/irida_api.py

```diff
@@ -366,12 +379,10 @@ def _get_analysis_result(self, analysis_submission_id):
         """
 
         if not self.analysis_submission_url:
-            logging.info("Requesting analysis submissions url.")
```

Comment:
Lets change these to `debug` statements instead of removing them. That way if work needs to be done in the future, we can simply change the output to debug mode and the logging will show up again.

Reply:
Done. Also added Developer Notes section in `README` to note that the progress bar will be disabled when on logging debugging mode.